src/applications/gi/components/vet-tec/VetTecCalculator.jsx

```diff
@@ -165,10 +165,12 @@ export class VetTecCalculator extends React.Component {
     const { showModal } = this.props;
     return (
       <div className="vads-l-row calculate-your-benefits">
-        <div className="medium-5 columns">{this.renderCalculatorForm()}</div>
-        <div className="medium-1 columns">&nbsp;</div>
-        <div className="medium-6 columns vads-u-margin-left--2 vads-u-margin-right--neg2">
-          <div className="your-estimated-benefits">
+        <div className="usa-width-five-twelfths medium-5 columns">
+          {this.renderCalculatorForm()}
+        </div>
+        <div className="usa-width-one-twelfth columns">&nbsp;</div>
```

Comment:
This is missing the `medium-1` class now, but the other two columns kept their `medium-*` classes. Maybe those classes should be removed from all of the columns, or `medium-1` should be added back to this one?

Reply:
Okay. I added it back in.